src/i18n/en.json

```diff
@@ -108,7 +108,10 @@
     "menuButtonLabel": "Profile menu",
     "profile": "Helsinki Profile",
     "signin": "Log in",
-    "signout": "Log out"
+    "signout": "Log out",
+    "menu": "Menu",
+    "skipToConten": "Skip to main content",
```

Comment:
`skipToContent` has a typo in it, this causes the translation not to work on the button.